Formula/hledger.rb

```diff
@@ -45,18 +45,30 @@ def install
     (buildpath/"../hledger-web").install resource("hledger-web")
     cd ".." do
       system "stack", "update"
-      system "stack", "init", "--resolver=lts-16.12"
-      system "stack", "install", "--system-ghc", "--no-install-ghc", "--local-bin-path=#{bin}"
+      (buildpath/"../stack.yaml").write <<~EOS
+        resolver: lts-16.27
+        compiler: ghc-8.10.3
+        compiler-check: newer-minor
+        packages:
+        - hledger-1.20.1
+        - hledger-lib
+        - hledger-ui
+        - hledger-web
+        extra-deps:
+        - pretty-simple-4.0.0.0
+        - prettyprinter-1.7.0
```

Comment:
Could you leave a brief comment before this that explains how these values are supposed to be updated? That way the next person who updates this formula can easily make the needed changes. Also:
```suggestion
        resolver: lts-16.27
        compiler: ghc-#{Formula["ghc"].version}
        compiler-check: newer-minor
        packages:
        - hledger-#{version}
        - hledger-lib
        - hledger-ui
        - hledger-web
        extra-deps:
        - pretty-simple-4.0.0.0
        - prettyprinter-1.7.0
```

Reply:
Thanks, done.

Reply:
Thanks. Sorry, one last thing, I think my suggestion may have obscured this in my previous comment:

>Could you leave a brief comment before this that explains how these values are supposed to be updated?

Reply:
Excellent; thank you!